client/tx/query.go

```diff
@@ -100,27 +96,23 @@ func ValidateTxResult(cliCtx context.CLIContext, res *ctypes.ResultTx) error {
 	return nil
 }
 
-func formatTxResult(cdc *codec.Codec, res *ctypes.ResultTx) (Info, error) {
+func formatTxResult(cdc *codec.Codec, res *ctypes.ResultTx) (sdk.ResponseFormat, error) {
 	tx, err := parseTx(cdc, res.Tx)
 	if err != nil {
-		return Info{}, err
+		return sdk.ResponseFormat{}, err
 	}
 
-	return Info{
-		Hash:   res.Hash,
-		Height: res.Height,
-		Tx:     tx,
-		Result: res.TxResult,
-	}, nil
+	return sdk.NewResponseResultTx(res, tx), nil
 }
 
-// Info is used to prepare info to display
-type Info struct {
-	Hash   common.HexBytes        `json:"hash"`
-	Height int64                  `json:"height"`
-	Tx     sdk.Tx                 `json:"tx"`
-	Result abci.ResponseDeliverTx `json:"result"`
-}
+//
+// // Info is used to prepare info to display
```

Comment:
Can we get rid of this?

Reply:
Was so shocked by how easy it was to pull this out I forgot to delete it :speak